File: PRAK501-2310817310008-MUHAMMADRIZKIRAMADHAN.c
#include <stdio.h>
int MaxBilangan(int, int, int, int);
int MaxBilangan(int a, int b, int c, int d){
   int maks = a;

   if (b > maks){
    maks = b;
   }
   if (c > maks){
    maks = c;
   }
   if (d > maks){
    maks = d;
   }
   return maks;
}

int main(){
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
    return 0;
}

File: PRAK501-2310817310008-MUHAMMADRIZKIRAMADHAN.py
def MaxBilangan(a, b, c, d):
    maks = a

    if (b > maks):
        maks = b
    if (c > maks):
        maks = c
    if (d > maks):
        maks = d
    return maks

a, b, c, d = map(int, input().split())
hasil = MaxBilangan(a, b, c, d)
print(hasil)
